graphql2/graphqlapp/rotation.go

```diff
@@ -375,3 +375,27 @@ func (m *Mutation) UpdateRotation(ctx context.Context, input graphql2.UpdateRota
 	}
 	return true, nil
 }
+
+func (a *Query) UpcomingHandoffTimes(ctx context.Context, input *graphql2.UpcomingHandoffTimesInput) ([]time.Time, error) {
+	var result []time.Time
+
+	if input.Hours <= 0 || input.Count <= 0 {
+		return result, errors.New("invalid upcoming handoff times input")
```

Comment:
This needs to be a field validation error for each. Use the validation helpers, otherwise, this is just going to come out as 500 unexpected error